site/content/blog/2022-08-22-the-param-miner-addon/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "The Param Miner Add-on"
+summary: >
+  The parameter discovery add-on for ZAP.
+images:
+- https://www.zaproxy.org/blog/2022-08-22-the-param-miner-addon/images/paramminer.png
```

Comment:
@psiinon do you think you can knock together a fun social preview image with zapbot & and a [mining pick](https://www.google.ca/search?q=mining+pick)?

Reply:
We had the mining pick at the beginning actually :P It was then changed to hard-mine-helmet! :) So our logo now is a mining helmet...

Reply:
Okay zapbot with a mining helmet :grinning:

Reply:
Looks like we need to change to a "mole" ... Sort of like the icon for golang :)... The pokemon Digglet would be good too :P

Reply:
I'll see what I can do... :)